src/main/java/com/github/snuk87/keycloak/kafka/KafkaEventListenerProviderFactory.java

```diff
@@ -61,6 +73,30 @@ public void init(Scope config) {
 			throw new NullPointerException("bootstrapServers must not be null");
 		}
 
+		if (saslUsername == null) {
+			throw new NullPointerException("saslUsername must not be null");
+		}
+
+		if (saslPassword == null) {
+			throw new NullPointerException("saslPassword must not be null");
+		}
+
+		if (saslProtocol == null) {
+			throw new NullPointerException("saslProtocol must not be null");
+		}
+
+		if (saslMechanism == null) {
+			throw new NullPointerException("saslMechanism must not be null");
+		}
+	
+		if (sslTruststoreLocation == null) {
+			throw new NullPointerException("sslTruststoreLocation must not be null");
+		}
+
+		if (sslTruststorePassword == null) {
+			throw new NullPointerException("sslTruststorePassword must not be null");
+		}
+
```

Comment:
This will break the deployment when ssl is not required and configured. The keycloak server will throw a NPE on startup and won't start at all.

Reply:
@SnuK87 I removed NullPointerExceptions for SASL + SSL related parameters. Let me know if there's anything else.